packages/sitecore-jss-react/src/components/FieldMetadata.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FunctionComponent } from 'react';
+
+/** The field metadata */
+export interface FieldMetadata {
+  contextItem?: FieldMetadataContextItem | null | undefined;
+  fieldId?: string | null | undefined;
+  fieldType?: string | null | undefined;
+  rawValue?: string | null | undefined;
+}
+
+/** The field's context item metadata  */
+export interface FieldMetadataContextItem {
+  id?: string | null | undefined;
+  language?: string | null | undefined;
+  revision?: string | null | undefined;
+  version?: number | null | undefined;
+}
+
+export interface FieldMetadataComponentProps {
+  htmlAttributes?: {
+    type: string;
+    chrometype: string;
+    className: string;
+  };
+  data: string;
+}
+
+const defaultAttributes = {
+  type: 'text/sitecore',
+  chrometype: 'field',
+  className: 'scpm',
+};
+
+export const FieldMetadataComponent: FunctionComponent<FieldMetadataComponentProps> = (
+  props: React.PropsWithChildren<FieldMetadataComponentProps>
+) => {
+  const attributes = {
+    ...defaultAttributes,
+    ...props.htmlAttributes,
```

Comment:
Trying to figure out. I think you don't really need to be able to customize _code_ tag attributes, they are the same for all fields, so we can just hardcode and use default attributes, _type_, _chrometype_, _className_, _kind_.